### ISSUE TITLE
Failed to import python plugin: official/Vector35_debugger: cannot import name 'Structure

### ISSUE THREAD

**Issue by EmbeddedBacon** [Vector35/deprecated_python_debugger#138]
Today updated BinaryNinja Personal 2.4.2974-dev, the Vector35 Debugger wasn't loading with the following message

Failed to import python plugin: official/Vector35_debugger: cannot import name 'Structure' from 'binaryninja' (C:\MyFiles\Apps\BinaryNinja\plugins\..\python\binaryninja\__init__.py)
Failed to load plugin: Vector35_debugger

Seems to be failing during the import phase found here during the `except` clause which was changed about 15 hours ago according to git blame

```
try:
	from binaryninja import Architecture, BinaryView, Symbol, SymbolType, Type, StructureBuilder, StructureType, FunctionGraphType, LowLevelILOperation, MediumLevelILOperation, core_ui_enabled, get_text_line_input, get_choice_input
	post3 = True
except:
	from binaryninja import Architecture, BinaryView, Symbol, SymbolType, Type, Structure, StructureType, FunctionGraphType, LowLevelILOperation, MediumLevelILOperation, core_ui_enabled, get_text_line_input, get_choice_input
	post3 = False
```

I am not sure if I need to install something else prior to installing this plugin or not.  This is Debugger v1.3.3 according to the "Manage Plugins" panel

**Comment by psifertex** [comment]
You can pull the latest version manually, we haven't done a release as we need to do more testing, but this should get you going in the meantime: 

TL;DR Just uninstall the plugin manager version and clone directly into your plugin folder. 

`git clone https://github.com/Vector35/debugger ~/.binaryninja/plugins/debugger`

(adjust as appropriate for your OS/user folder location)

**Comment by EmbeddedBacon** [comment]
Thanks, @psifertex, I will give it a try

**Comment by psifertex** [comment]
I only did a cursory amount of testing but it appeared to work at least similarly to before so definitely let me know if you run into any other refactor issues!

**Comment by EmbeddedBacon** [comment]
Thanks, I finally found where its installation directory should be, at least in my case.

`%APPDATA%\Roaming\Binary Ninja\repositories\official\plugins\`

I get the nice message once again

`Loaded python3 plugin 'Vector35_debugger'`

**Comment by psifertex** [comment]
Yeah, that location is documented: https://docs.binary.ninja/getting-started.html#user-folder

And you can also just use the "Tools" / "Open Plugin Folder" menu option.

I'm going to close this one since the next stable release will automatically solve this and the fix is available for folks that do a manual install.

**Comment by EmbeddedBacon** [comment]
For me, if I go the route of "Tools" | "Open Plugin Folder" it opens `%APPDATA%\Roaming\Binary Ninja\plugins`, which isn't really correct in my case as it is empty.  My plugins are stored under the following

%APPDATA%\Roaming\Binary Ninja\repositories\community\plugins
or
%APPDATA%\Roaming\Binary Ninja\repositories\official\plugins

Not sure what the difference is, but I will chalk it up to Windows  ;-)

**Comment by psifertex** [comment]
Please do not put plugins in the other locations. When you manually install them you want to use the first folder that the Tools menu opens. Any other location is incorrect. Nothing is in there because you haven't installed any other plugins manually yet, that's ok! 😉
